_modules/constants.js

```diff
@@ -0,0 +1,21 @@
+
+/**************************************************************************************************
+ * Variables
+ **************************************************************************************************/
+
+/** Standardized error message for an invalid package version string */
+const _errorInvalidVersionString = "Invalid package version string (see https://docs.microsoft.com/en-us/nuget/concepts/package-versioning)";
+
+/** Standardized error message when an error occurs while reading csproj files */
+const _errorReadingCsprojFiles = "There was an error reading csproj files.";
+
+/** Semver regex pattern for validating version number (see https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) */
+const _versionRegexPattern = /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/;
+
+
+/**************************************************************************************************
+ * Exports
+ **************************************************************************************************/
+exports.ERROR_INVALID_VERSION_STRING = _errorInvalidVersionString;
```

Comment:
Any reason to not just do `export const ERROR_INVALID_VERSION_STRING = "Invalid ...";`? Saves the extra lines

Reply:
Nope, no real reason. Guess it was just my first instinct based on how I structured some exports for earlier versions of the modules, I can update this though!